src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.for {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+  padding: 23px 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 900;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+input:hover {
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input:focus {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  color: #3d4e61;
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.bigger {
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: url(images/Search.svg) top 25px left 26px;
+  background-repeat: no-repeat;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background-size: 19px;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+}
+
+.smaller {
+  height: 42px;
+  background: url(images/Search.svg) no-repeat left 13px top 15px;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  background-size: 11px;
+  margin-top: 20px;
+  padding-left: 33px;
+  padding-top: 12px;
+  padding-bottom: 10px;
+}
+
+.bigger:hover {
+  background: url(images/Search.svg) top 25px left 26px;
+  background-repeat: no-repeat;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+}
+
+.bigger:focus {
+  font-size: 16px;
+  line-height: 22px;
+  background: url(images/Search.svg) top 25px left 26px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  background-size: 19px;
+}
+
+.smaller:hover {
+  font-size: 14px;
+  line-height: 19px;
+  background: url(images/Search.svg) left 13px top 15px;
+  background-repeat: no-repeat;
+  background-size: 11px;
+}
+
+.smaller:focus {
+  background: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  background-position: left 13px top 15px;
+  background-size: 11px;
+}
+
+.bigger::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.smaller::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  color: #3d4e61;
+  font-weight: 400;
```

Comment:
you can add this style for .input::placeholder with font-size 16px and for .smaller::placeholder fix font-size to font-size 14px